srcpkgs/diction/patches/0001-fix-cross.patch

```diff
@@ -0,0 +1,48 @@
+From 27fb43ea79be3caefd109a082f4744efb925605c Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens@artixlinux.org>
+Date: Sun, 3 Jan 2021 08:44:42 -0600
+Subject: [PATCH] fix cross
+
+---
+ configure.in | 8 ++++----
+ 1 file changed, 4 insertions(+), 4 deletions(-)
+
+diff --git configure.in configure.in
+index 1e4dac4..44ee54f 100644
+--- configure.in
++++ configure.in
+@@ -34,7 +34,7 @@ AC_CHECK_FUNCS(strerror)
+ AC_SEARCH_LIBS(regcomp, [regex pcre])
+ 
+ AC_MSG_CHECKING(for broken realloc)
+-AC_TRY_RUN([#include <sys/types.h>
++AC_COMPILE_IFELSE([#include <sys/types.h>
+ #include <signal.h>
+ #include <stdlib.h>
+ 
+@@ -49,7 +49,7 @@ int main(void)
+   realloc((void*)0,1);
+   exit(0);
+ }
+-],AC_MSG_RESULT(no),AC_DEFINE(BROKEN_REALLOC) AC_MSG_RESULT(yes))
++],AC_MSG_RESULT(no),AC_DEFINE([BROKEN_REALLOC, [1], [Define broken realloc]]) AC_MSG_RESULT(yes))
```

Comment:
The brokenness of `realloc` couldn't be determined by compiling and linking.
We need to run it. Or as the old code does, specify what we know.